pkg/actors/actors.go

```diff
@@ -76,7 +76,7 @@ var (
 	log = logger.NewLogger("dapr.runtime.actor")
 
 	ErrIncompatibleStateStore        = errors.New("actor state store does not exist, or does not support transactions which are required to save state - please see https://docs.dapr.io/operations/components/setup-state-store/supported-state-stores/")
-	ErrReminderOpActorNotHosted      = errors.New("operations on actor reminders are only possible on hosted actor types")
+	ErrReminderOpActorNotHosted      = errors.New("operations on actor reminders are only possible on hosted actor types. Check that the actor id/type is correct and hosted by the expected instance")
```

Comment:
Only type is relevant in this context afaik

Reply:
In my client code I swapped the actor id/type accidentally like this:
```
...
	err = client.RegisterActorReminder(ctx, &dapr.RegisterActorReminderRequest{
		ActorType: "playerActorType",
		ActorID:   actorID,
		Name:      "healthReminder",
		DueTime:   "1s",
		Period:    "2s",
		Data:      []byte(`"Reminder triggered"`),
	})
...
	err = client.UnregisterActorReminder(ctx, &dapr.UnregisterActorReminderRequest{
		ActorType: actorID, <------------------HERE
		ActorID:  "playerActorType", <------------------HERE
		Name:      "healthReminder",
	})
...
```

From a user perspective, it would have sped up my debugging bc I assumed it was a bug and missed that I swapped the id/type

Reply:
I agree that only the type is relevant for this error though.